Trell/CliCommands/InitCommand.cs

```diff
@@ -99,16 +111,20 @@ public async override Task<int> ExecuteAsync(CommandContext context, InitCommand
         );
 
         if (!shouldSkipExample) {
+            const string INVALID_INPUT_MSG = "Please use only lowercase letters (a-z), numbers (0-9), and underscore (_)";
+
             var username = settings.Username ?? AnsiConsole.Prompt(
-                new TextPrompt<string>("Please provide a username")
+                new TextPrompt<string>("Please provide a username, using only lowercase letters (a-z), numbers (0-9), and underscore (_)")
```

Comment:
These prompt lines end up being kind of long. How does the flow end up if you print out a "Please use only lowercase letters (a-z), numbers (0-9), and underscore (_) for the username and new worker name" message and then keep the `TextPrompt` the same as before?